zerver/tests/test_realm_export.py

```diff
@@ -95,6 +95,26 @@ def test_realm_export_rate_limited(self) -> None:
                                          event_type=RealmAuditLog.REALM_EXPORTED,
                                          event_time=timezone_now()))
         RealmAuditLog.objects.bulk_create(exports)
-
         result = public_only_realm_export(self.client_post, admin)
         self.assert_json_error(result, 'Exceeded rate limit.')
+
+    def test_public_exports_get(self) -> None:
+        admin = self.example_user('iago')
+        self.login(admin.email)
+        time = timezone_now()
+
+        RealmAuditLog.objects.create(realm=admin.realm,
+                                     event_type=RealmAuditLog.REALM_EXPORTED,
+                                     event_time=time)
+
+        result = self.client_get('/json/export/realm')
+        self.assert_json_success(result)
+        export_dict = result.json()['public_exports']
+
+        self.assertEqual(export_dict[0]['realm'], 'Zulip Dev')
+        self.assertEqual(export_dict[0]['event_time'], time.ctime())
+        self.assertEqual(export_dict[0]['acting_user'], 'Iago')
+        self.assert_length(export_dict,
+                           RealmAuditLog.objects.filter(
+                               realm=admin.realm,
+                               event_type=RealmAuditLog.REALM_EXPORTED).count())
```

Comment:
Rather than adding a new test with fake RealmAuditLog data, you should just extend an existing test to do the `client_get` verification; that's more robust to future changes since we're testing the structures we actually create.

Reply:
Cool, I appended this check to both of the endpoint tests.